corehq/apps/enterprise/enterprise.py

```diff
@@ -383,3 +388,68 @@ def _get_individual_export_rows(self, exports, export_line_counts):
             )
 
         return rows
+
+
+class EnterpriseCommCareVersionReport(EnterpriseReport):
+    title = _('CommCare Client Version Compliance')
+    metric = _('%% of mobile workers on the latest commcare client version ')
```

Comment:
Metric may be confusing as it may suggest Kissmetrics or Google Analytics. We may want to rename it? Description, subtitle, etc.

Reply:
5f91e2b1109de4e49d6cd9cb41b39fff2d60ed0e